RE: Fix UI Layout

Adyel opened the pull request:
- Fixed conflict
- Fixed code formatting
- Added UI Layout

Adyel commented:
@mominul  Please do a final check before pushing to the main branch.

mominul commented on src/frontend/SettingsDialog.ui:
```diff
@@ -1,178 +1,133 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ui version="4.0">
```
![settings_cutoff](https://user-images.githubusercontent.com/9459891/43120519-4b2f2762-8f3c-11e8-839a-14548a82ed77.png)
The text is being cut off.

mominul commented on src/frontend/AboutDialog.ui:
```diff
@@ -1,869 +1,2568 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
![too_much_about](https://user-images.githubusercontent.com/9459891/43120616-ab9ebc3e-8f3c-11e8-84b0-63f822b4b883.png)
Too much spaces!

Adyel replied:
This is OLD School one, I'm not gonna change it's layout for now. It requires some rework.

mominul commented on src/frontend/AboutFile.ui:
```diff
@@ -6,153 +6,152 @@
    <rect>
```
![about_layout_mistakes](https://user-images.githubusercontent.com/9459891/43120682-ea106c10-8f3c-11e8-946d-a8816354272e.png)
That `Ok` button is too much wide :)

mominul commented on src/frontend/AutoCorrectDialog.ui:
```diff
@@ -1,228 +1,167 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
![autocorrect_mistakes](https://user-images.githubusercontent.com/9459891/43120741-2618d440-8f3d-11e8-95ab-66826eec201d.png)
That two text boxes are unequal in size. `Clear` button is also quite inconsistent.

Adyel commented:
@mominul 
Fixed the look

**AboutFile.ui**
![image](https://user-images.githubusercontent.com/4653623/43244222-cc657cec-90cb-11e8-8b08-6cb6b7f29414.png)


**SettingsDialog.ui**
This looked fine on my End. After your report I added padding just in case. Please, recheck if it is fixed.
![image](https://user-images.githubusercontent.com/4653623/43244281-f955f0b0-90cb-11e8-9dc4-a71dc4442009.png)

**AutoCorrectDialog.ui**
![image](https://user-images.githubusercontent.com/4653623/43244330-32c4b2dc-90cc-11e8-989f-531457f79ab2.png)

Adyel commented:
@mominul  So, looks ok? Merge?

mominul commented:
Sorry I haven't tested out the changes yet! 😫

mominul commented on src/frontend/SettingsDialog.ui:
```diff
@@ -13,132 +13,139 @@
   <property name="windowTitle">
```
![settings_flaw](https://user-images.githubusercontent.com/9459891/43338196-82e2bc6a-91f7-11e8-873f-c1668c43f07e.png)
The padding didn't work. There are now two lines which are being cut off.

Adyel replied:
Hmm, That's weird. This window supposes to have a fixed width. It is showing way smaller in your screenshot.

Adyel replied:
@mominul Can you come to discord.

mominul commented:
Merging this now. Will have further discussions at #56 . Thanks for the work @Adyel !
